package.json

```diff
@@ -11,7 +11,7 @@
   ],
   "scripts": {
     "build": "node-gyp build",
-    "tsc": "tsc -p node && cp node/*.d.ts node/dist",
```

Comment:
Looks like `tsc -p node --declarationDir node/dist` is close enough (possibly better, even). `lint` and `format` are just going to suffer, but that's fine.

Reply:
Thanks for pointing that out `--declarationDir` seems much better than the possible alternatives here. And yeah I was content with leaving lint and format broken on Windows.

Reply:
Ugh, I was wrong about this. We already had the TypeScript-generated .d.ts files going into `dist`; it's libsignal_client.d.ts that's missing.

Reply:
You've got it manually in the distribution script, so I guess it's just something I'll have to handle if I want `yarn pack` to replace that script.